src/app/content/highlights/components/ShowMyHighlights.spec.tsx

```diff
@@ -0,0 +1,258 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-dom/test-utils';
+import { Provider } from 'react-redux';
+import renderer, { act } from 'react-test-renderer';
+import createTestStore from '../../../../test/createTestStore';
+import createMockHighlight from '../../../../test/mocks/highlight';
+import { renderToDom } from '../../../../test/reactutils';
+import { receiveFeatureFlags } from '../../../actions';
+import { receiveUser } from '../../../auth/actions';
+import { User } from '../../../auth/types';
+import MessageProvider from '../../../MessageProvider';
+import { locationChange } from '../../../navigation/actions';
+import { Store } from '../../../types';
+import { assertWindow } from '../../../utils';
+import { content } from '../../routes';
+import { openMyHighlights, receiveHighlights } from '../actions';
+import { highlightingFeatureFlag, highlightStyles } from '../constants';
+import { summaryIsLoading } from '../selectors';
+import HighlightsPopUp from './HighlightsPopUp';
+import { PopupBody } from './HighlightStyles';
+import { ShowMyHighlightsBody } from './ShowMyHighlightsStyles';
+
+describe('MyHighlights button and PopUp', () => {
+  let store: Store;
+  let user: User;
+  let highlight1: ReturnType<typeof createMockHighlight>;
+  let highlight2: ReturnType<typeof createMockHighlight>;
+  let window: Window;
+
+  beforeEach(() => {
+    store = createTestStore();
+    user = {firstName: 'test', isNotGdprLocation: true, uuid: 'some_uuid'};
+
+    store.dispatch(receiveFeatureFlags([highlightingFeatureFlag]));
+
+    highlight1 = createMockHighlight('asdf');
+    highlight2 = createMockHighlight('lkjh');
+    window = assertWindow();
+  });
+
+  it('opens highlights pop up with user Highlights', async() => {
+    act(() => {
+      store.dispatch(receiveUser(user));
+      store.dispatch(receiveHighlights([
+        {
+          ...highlight1.serialize().data,
+          note: 'adsf',
+          style: highlightStyles[0].label,
+        },
+        {
+          ...highlight2.serialize().data,
+        },
+      ]));
+    });
+
+    const component = renderer.create(<Provider store={store}>
+      <MessageProvider>
+        <HighlightsPopUp></HighlightsPopUp>
```

Comment:
you're still rendering the `HighlightsPopUp` component, you should render the `ShowMyHighlights` component directly to simplify these tests